ClientLib/src/main/java/com/o3dr/android/client/apis/drone/ExperimentalApi.java

```diff
@@ -28,6 +32,15 @@ public static void triggerCamera(Drone drone) {
         drone.performAsyncAction(new Action(ACTION_TRIGGER_CAMERA));
     }
 
+    public static void setROI(Drone drone, double latitude, double longitude, double altitude) {
```

Comment:
@Teemperor you can use a [`LatLongAlt`](https://github.com/DroidPlanner/DroneKit-Android/blob/d6c370bd55784d353b0567cc5e30e773173ebe34/ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/LatLongAlt.java) object here instead as it's parcelable.